caim_base/templates/animal/view.html

```diff
@@ -117,9 +117,9 @@ <h4><a href="{{animal.awg.get_absolute_url}}">{{ animal.awg.name }}</a></h4>
     </div>
   </div>
 
-
-
-
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"
```

Comment:
JQuery is already included. Do we need this again? Its currently in the footer of the page, so it should work if you use dom ready correctly in the above code.

Reply:
I noticed a console error. JQuery should always be the topmost tag, else it causes some issues often.

Reply:
If you view the code, you see:

![Screenshot 2022-10-31 at 15 41 07](https://user-images.githubusercontent.com/149461/199048807-db76e641-ed58-4bb6-8338-d72f4a56812c.png)

jquery is included twice

Reply:
I see your point. One on ethe wrapper and then the one I added to the view template. WHat do yo suggest?